lib/embed/media_tag.rb

```diff
@@ -19,22 +19,20 @@ def initialize(ng_doc, viewer)
     attr_reader :ng_document, :purl_document, :request, :viewer
 
     def build_markup
-      file_index = 0
       purl_document.contents.each do |resource|
-        label = resource.description
         next unless SUPPORTED_MEDIA_TYPES.include?(resource.type.to_sym)
+        label = resource.description
         resource.files.each do |file|
           label = file.title if label.blank?
           ng_document.send(
             resource.type.to_sym,
             'data-file-label': label,
             'data-slider-object': file_index,
+            'data-src': streaming_url_for(file, :dash),
```

Comment:
The reason there is an `ENABLED_STREAMING_TYPES` is because it wasn't immediately obvious (and it still not solidified) that the MPEG-DASH url will be used in child `<source>` element  or as the `src` attribute of the `<video>` element itself.  Having this as an array that can be configured/modified is an attempt to make this code open to new formats or changing how we handle existing formats.

I agree that it should be in fewer places though. This has mainly come out of a few different features coming together where we now have a more cohesive direction for configuration.

Would you view it as an improvement in this regard if we had a single `streaming` configuration in `Settings.yml` that included the pertinent information about the enabled types, suffixes, and mimetypes?